views/templates/Lists/index.tpl

```diff
@@ -13,28 +13,32 @@
 </div>
 
 <div class="data-container-wrapper flex-container-remainder">
+    <?php
+        $editListTitle = __('Edit this list');
+        $deleteListTitle = __('Delete this list');
+    ?>
     <?php foreach (get_data('lists') as $i => $list): ?>
-    <section id='list-data_<?= $list["uri"] ?>' class="data-container list-container">
+    <section id="list-data_<?= $list['uri'] ?>" class="data-container list-container">
         <header class="container-title">
             <h6><?= _dh($list['label']) ?></h6>
         </header>
-        <div class="container-content" id='list-elements_<?= $list['uri'] ?>'>
+        <div class="container-content" id="list-elements_<?= $list['uri'] ?>" data-total-count="<?= $list['totalCount'] ?>">
```

Comment:
- [ ]  Fix this - use this to show total number of elements

Reply:
Ping @adriloma

Reply:
Hi @shpran , I implemented it on line42: 
```
<span><?php echo $list['totalCount'] ?> elements</span>
```
So I think I can remove this data attribute.